src/components/notifications/content-notification/README.md

```diff
@@ -0,0 +1,22 @@
+# ContentNotification
+
+#### Description
+
+An in-place notification panel.
+
+## Usage
+
+```js
+import { ContentNotification } from '@commercetools-frontend/ui-kit';
+
+<ContentNotification type="error">
+  Something went wrong here!
+</ContentNotification>;
+```
+
+#### Properties
+
+| Props      | Type     | Required | Values                                | Default | Description                                       |
+| ---------- | -------- | :------: | ------------------------------------- | ------- | ------------------------------------------------- |
+| `children` | `node`   |    -     | -                                     | -       | Message to be shown inside a notification panel.  |
+| `type`     | `string` |    -     | `success`, `info`, `warning`, `error` | -       | Type of notification, determining its appearance. |
```

Comment:
@mariabarrena Can we include some guidelines (Dos & Don'ts) here on when to use `ContentNotification`s over `SideNotification`s ? What's your take on it? :)